test/test_torch.py

```diff
@@ -10842,18 +10858,21 @@ def test_eye(self, device):
 
     def test_addcmul(self, device):
         def rand_tensor(size, dtype, device):
-            if dtype.is_floating_point:
+            if dtype.is_floating_point or dtype.is_complex:
                 return torch.rand(size=size, dtype=dtype, device=device)
             if dtype == torch.uint8:
                 return torch.randint(1, 5, size=size, dtype=dtype, device=device)
             else:
                 return torch.randint(-5, 5, size=size, dtype=dtype, device=device)
 
         for dtype in torch.testing.get_all_math_dtypes(device):
+            # TODO: update this after torch.isclose is landed for complex
+            if dtype.is_complex:
```

Comment:
I liked the thing you did earlier where you expected an error; that will help remind you to update this when isclose starts working

Reply:
yeah makes sense updated